doc/examples.md

```diff
@@ -130,51 +132,19 @@ Or implicitly when following an object reference. In both cases the access to da
 Sometimes it is necessary or useful to store additional data that is not directly foreseen in the EDM.
 This could be configuration parameters of simulation jobs, or parameter descriptions like cell-ID encoding etc. PODIO currently allows to store such meta data in terms of a `GenericParameters` class that
 holds an arbitrary number of named parameters of type `int, float, string` or vectors if these.
-Meta data can be stored and retrieved from the `EventStore` for runs, collections and events via
-the three methods:
-```cpp
-virtual GenericParameters& EventStore::getRunMetaData(int runID);
```

Comment:
should we add a small explanation that one just creates a frame if there is a metadata use case? Just to explain a bit more the frame concept

Reply:
for a non-default one where putting a parameter is not enough

Reply:
That can go to a separate PR